.gitpod.yml

```diff
@@ -0,0 +1,14 @@
+tasks:
+  - name: Setup and Build
+    before: |
+      # Download and Install Java 17.0.6-tem
+      yes Y | sdk install java 17.0.6-tem
+
+      # Download and Install Maven 3.9.0
+      yes Y | sdk install maven 3.9.0
+    init: |    
+      #Build using Maven
+      mvn package -Dmaven.test.skip
+      
+      # Install Yarn dependencies
+      yarn install
```

Comment:
Technically, this is done during the `generate-sources` phase of Maven (https://github.com/jenkins-infra/plugin-health-scoring/blob/main/pom.xml#L340-L349). I'm not sure this is strictly required as after the `mvn package`, the users would have downloaded the Yarn dependencies.

Reply:
Thank you for letting me know